### DISCUSSION TITLE
Puedo usar el FAB button para iniciar un flujo de una submáquina de estados?

### DISCUSSION THREAD

**Discussion by jdurandpaz** [Cotalker/documentation#196] [2. Questions and Answers] [answered]
En la SM de proyectos tenemos una sub máquina de tareas, me gustaría saber si puedo usar el FAB button con required survey para crear tareas en la sub máquina.

Saludos

**Comment by gmoplaza** [comment, marked as answer]
Hola @jdurandpaz por ahora no se puede realizar lo pides, pero estamos trabajando justamente en la funcionalidad que necesitas.

Saludos

**Reply by acedward** [reply]
@jdurandpaz si esto funcionara, como te gustaríá que se pudiera elegir la tarea padre?

**Reply by jdurandpaz** [reply]
podría ser elegir que flujo quiero dar inicio con la acción

**Reply by guillevalin** [reply]
@jdurandpaz esto vendrá en la nueva funcionalidad de actions, atento al próximo release!
